index.html

```diff
@@ -1629,6 +1629,53 @@ <h4>DIDCommMessaging</h4>
 
       </section>
 
+      <section>
+        <h4>CredentialRegistry</h4>
+        <p>The CredentialRegistry endpoint allows publication of a dedicated service endpoint in a DID document through which verifiable credentials can be queried. Each registry endpoint is a REST endpoint. When a GET request is sent to the URI formed by appending the <code>credentialSubject.id</code> as a URL-encoded string to the given endpoint URI, the registry MUST return an array of verifiable credentials associated with the subject ID. A sample registry endpoint can be found <a href="https://ssi.eecc.de/api/registry/swagger/#/Credentials/get_api_registry_vcs__id_">here</a>.</p>
+        <p class="note">
+          Verifiable credential registries are supposed to hold credentials that are publicly accessible by default, e.g., for product passports on a product type level. An additional authentication for limiting access to certain credentials is currently under development.
+        </p>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/swagger/#/Credentials/get_api_registry_vcs__id_">Verifiable Credential Registry</a>
+              </td>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/context/credentialregistry">Verifiable Credential Registry</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of service and serviceEndpoint properties">
+{
+  ...
+  "service": [
+    {
+      "id": "did:example:123#vcregistry-1",
+      "type": "CredentialRegistry",
+      "serviceEndpoint": {
+        "registries": ["https://registry.example.com", "https://identity.foundation/vcs"]
+      }
+    },
+    {
+      "id": "did:example:123#vcregistry-2",
+      "type": "CredentialRegistry",
+      "serviceEndpoint": "https://ssi.eecc.de/api/registry/vcs/"
```

Comment:
404, is there a better example? or use `example.com`

Reply:
Ah, when calling it plain, the request parameter is missing. It should be https://ssi.eecc.de/api/registry/vcs/{credentialSubject.id}
In our case the additional HTTP Header `'Accept': 'application/ld+json,application/json'` is needed, because browser (text/html) requests get redirected to the verifier service which then retrieves and verifies the credentials.

Reply:
Example
```
curl -X 'GET' \
  'https://ssi.eecc.de/api/registry/vcs/https%3A%2F%2Ftest.de%2Ftest1' \
  -H 'accept: application/ld+json, application/json'
```

Reply:
feel free to include the curl example if you like.

Reply:
```suggestion
      "serviceEndpoint": "https://ssi.eecc.de/api/registry/vcs/{credentialSubject.id}"
```